src/components/Sticky/index.stories.js

```diff
@@ -0,0 +1,102 @@
+import { storiesOf } from '@storybook/react'
+import React, { PureComponent } from 'react'
+import Sticky from '.'
+import CodeExample from '../../utils/CodeExample'
+import DocHeader from '../../utils/DocHeader'
+import DocSection from '../../utils/DocSection'
+import withAddons from '../../utils/withAddons'
+import Button from '../Button'
+import Select from '../Select'
+
+const positionOptions = [
+  { value: 'top', label: 'Top' },
+  { value: 'bottom', label: 'Bottom' },
+]
+
+class StickyStory extends PureComponent {
+  state = {
+    position: 'top',
+    width: '100%',
+  }
+
+  set = name => value =>
+    this.setState({
+      [name]: value,
+    })
+
+  render () {
+    const { position } = this.state
+    const containerStyle = {
+      paddingTop: '60px',
+      paddingBottom: '1000px',
+    }
+    const navbarStyle = {
```

Comment:
If you want, you can move the example styles into the "upside down" scss file as well - it's where we've put any styles that aren't needed for mc-components, but _are_ needed for the documentation in storybook.  We called it upside down when stranger things had just come out... :D 

https://github.com/yankaindustries/mc-components/blob/develop/.storybook/_upside-down.scss